lib/views/pages/screens/profile_screen.dart

```diff
@@ -4,6 +4,134 @@ class ProfileScreen extends StatelessWidget {
   const ProfileScreen({Key? key}) : super(key: key);
 
   @override
-  Widget build(BuildContext context) =>
-      const Center(child: Text('ProfilePage'));
+  Widget build(BuildContext context) => Scaffold(
+          backgroundColor: const Color.fromARGB(255, 40, 40, 40),
+          body: SafeArea(
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: <Widget>[
+                  const SizedBox(
+                    height: 125,
+                    width: 200,
+                  ),
+                  const CircleAvatar(
+                    radius: 55,
+                    backgroundImage: AssetImage('images/deltaLogoGreen.png'),
+                    backgroundColor: Colors.black,
+                  ),
+                  const SizedBox(
+                    height: 15,
+                    width: 200,
+                  ),
+                  const Text(
+                    'Student Name',
+                    style: TextStyle(
+                      fontFamily: 'SourceSansPro',
+                      fontSize: 25,
+                      color: Colors.white,
+                      shadows: <Shadow>[ Shadow(
+                    offset: Offset(1.0, 1.0),
+                    blurRadius: 1.0,
+                    color: Colors.white
+                  )
+                ]
+                    ),
+                  ),
+                  const SizedBox(
+                    height: 5,
+                    width: 200,
+                  ),
+                  const Text(
+                    '106120011@nitt.edu',
+                    style: TextStyle(
+                      fontSize: 14,
+                      fontFamily: 'SourceSansPro',
+                      color: Color.fromARGB(255, 2, 200, 20)
+                    ),
+                  ),
+                  SizedBox(
+                    height: 20.0,
+                    width: 200,
+                    child: Divider(
+                      color: Colors.teal[100],
+                    ),
+                  ),
+                  const SizedBox(
```

Comment:
Create a custom widget for the card and iterate over it for all the 4 cards

reduces hardcoding to increase code maintainability